pytorch_toolkit/object_detection/tests/train_tests_text_detection.py

```diff
@@ -48,8 +48,8 @@ def setUp(self):
                                         'annotation.json')
             assert replace_text_in_file(self.configuration_file, 'IC13TEST.json',
                                         'annotation.json')
-            assert replace_text_in_file(self.configuration_file, "evaluation = dict(interval=1, metric='f1')",
-                                        '')
+            assert replace_text_in_file(self.configuration_file, "evaluation = dict(interval=",
```

Comment:
What if we set metrics in evaluation as ['bbox', 'f1'] without any modification here

Reply:
In train_tests I use mAP metric to check the the accuracy, so I simply remove the evaluation line from config, because mAP is calculated by default. 
I changed this line as far as now line from config contains parameter ```score_thr=0.25``` and previous version of the command could not replace the line, since it simply could not find it. For this reason even if I set metrics in evaluation as [`bbox`, `f1`] it still will be necessary to change this line

Reply:
Not clear for me why changes in this line will be required. But one more thing that if you don't add ['bbox', 'f1'] then mAP is not calculated during training. I think it is also useful.

Reply:
I mean I would simply remove line here

Reply:
Going forward, if you don't do what I'm asking I won't be able to refactor your test as following https://github.com/openvinotoolkit/training_extensions/blob/model_template/pytorch_toolkit/object_detection/tests/train_tests_text_detection.py